### ISSUE TITLE
Some stickers not found

### ISSUE THREAD

**Issue by tabam1** [Step7750/node-csgo-cdn#21]
Some sticker return nothing, e.g. this:
`"material": "chicken_capsule/BoneHead_pack01" / "name": "Sticker | Bonehead" `
or
`"material": "chicken_capsule/NestEgg_pack01" / "name": "Sticker | Nest Egg"`

**Comment by tabam1** [comment]
Will be fixed ?

**Comment by Step7750** [comment]
Currently no ETA nor have I investigated this, feel free to submit a PR for this.

It is likely that Valve had casing/spelling difference between the game files and actual resource.

**Comment by centrual** [comment]
Skins not found:
```
Atlanta 2017 Challengers (Holo-Foil)
Berlin 2019 Legends (Holo-Foil)
Berlin 2019 Minor Challengers (Holo-Foil)
Berlin 2019 Minor Challengers (Holo-Foil)
Berlin 2019 Returning Challengers (Holo-Foil)
Cologne 2016 Legends (Holo-Foil)
DreamHack 2014 Legends (Holo-Foil)
ESL One Katowice 2015 Challengers (Holo-Foil)
ESL One Katowice 2015 Legends (Holo-Foil)
Katowice 2019 Legends (Holo-Foil)
Katowice 2019 Minor Challengers (Holo-Foil)
Katowice 2019 Returning Challengers (Holo-Foil)
Krakow 2017 Legends (Holo-Foil)
London 2018 Legends (Holo-Foil)
London 2018 Minor Challengers (Holo-Foil)
London 2018 Returning Challengers (Holo-Foil)
MLG Columbus 2016 Challengers (Holo-Foil)
MLG Columbus 2016 Legends (Holo-Foil)
Sticker | Boom(Foil)
Sticker | Double Dip
Sticker | Heads Up
Sticker | Hot Wings
Sticker | Move It(Foil)
Sticker | Nest Egg
Sticker | Roosty Boosty
Sticker | What What
```

I trying to fix this problem...

**Comment by denkomanceski** [comment]
Actually, the issue is the following:

![image](https://user-images.githubusercontent.com/10501247/82160774-fe2bfa00-9897-11ea-9617-d5b9c7033747.png)


Some of the stickers are not in `resource/flash/econ/stickers` but `resource/flash/econ/patches`

Example: `resource/flash/econ/patches/casehlalyx/patch_hlalyx_vort_large.png`

**Comment by denkomanceski** [comment]
Also, some other items had a mix of upper/lowercase letter in their sticker_material, so I had to .lowerCase() it in order to find it

Maybe those are the missing ones, I didnt check their names sry @centrual

**Comment by danieldspx** [comment]
> Actually, the issue is the following:
> 
> ![image](https://user-images.githubusercontent.com/10501247/82160774-fe2bfa00-9897-11ea-9617-d5b9c7033747.png)
> 
> Some of the stickers are not in `resource/flash/econ/stickers` but `resource/flash/econ/patches`
> 
> Example: `resource/flash/econ/patches/casehlalyx/patch_hlalyx_vort_large.png`

#37 will fix this issue.

**Comment by Step7750** [comment]
#37 is merged and released in v1.2.0, thanks @danieldspx!
